.gitignore

```diff
@@ -27,6 +27,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+venv/
```

Comment:
Thanks for doing this. I use virtualenv-wrapper, so my venv folder in not in the project folder. I just didn't want to have to explain another step.